tree/dataframe/inc/ROOT/RDF/RSample.hxx

```diff
@@ -40,6 +51,7 @@ class RSample {
     * They can contain the globbing characters supported by TChain. See TChain::Add for more information.
     */
    std::vector<std::string> fFileNameGlobs;
+   /// An instance of the RMetaData class.
    RMetaData fMetaData;
 
    unsigned int fSampleId{0}; // global sample index, set inside of the RDatasetSpec
```

Comment:
To make this one also show up in Doxygen and make it consistent with the other docstrings:
```suggestion
   /// global sample index, set inside of the RDatasetSpec
   unsigned int fSampleId{0};
```